src/test/java/com/wcc/platform/service/CmsServiceTest.java

```diff
@@ -33,7 +32,7 @@ class CmsServiceTest {
   void setUp() {
     MockitoAnnotations.openMocks(this);
 
-    service = new CmsService(objectMapper, footerRepository, landingPageRepository);
+    service = new CmsService(objectMapper, pageRepository);
```

Comment:
Do we need to add a test to use mock pageRepository to fetch the data from repository + if not found then fallback ?

Reply:
It is missing the test for the success case using the surrealDB, now we are only testing the fallback behaviour. I will do that in another issue: [#128](https://github.com/Women-Coding-Community/wcc-backend/issues/128)